docs/src/glossary.rst

```diff
@@ -39,22 +37,35 @@ Here are some definitions of the specific terms used in the PyMoDAQ documentatio
     linear "positions", for instance data taken at random time intervals. Data can therefore have two distributions:
     **uniform** or **spread**.
 
-  Signal
-    Signal and Navigation is a term taken from the hyperspy package vocabulary. It is useful when dealing with
-    multidimensional data.
-    Imagine data you obtained from a camera (256x1024 pixels) during a linear 1D scan of one actuator (100 steps).
-    The final shape of the data would be (100, 256, 1024). The first dimension corresponds to a Navigation axis
-    (the scan), and the rest to Signal axes (the real detector's data).
-    The corresponding data has a dimensionality of DataND and a representation of (100|256,1024).
+  DataSource
+    Enum informing about the source of the data object, for instance raw from a  detector or processed from
+    mathematical functions (from ROI, ...)
 
-  Navigation
-    See above.
+  Detector
+    Any instrument generating data to be recorded
+
+  dte
+    Short name for ``DataToExport`` object
 
   dwa
     Short name for ``DataWithAxes`` object
 
-  dte
-    Short name for ``DataToExport`` object
+  Module
+     A module in the python sense is an importable object either a directory containing an *__init__.py* file or a
+     python file containing data, functions or classes.
+
+.. note::
+    If there is code that can be executed within your module but you don't want it to be executed when importing,
+    make sure to protect the execution using a : ``if __name__ == '__main__':`` clause.
+
+.. glossary::
+
+  Navigation
+    See above.
```

Comment:
should be put just below the Signal term

Reply:
This change happened because i reordered alphabetically the terms.
I added a link to the Signal term.